jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/list/AbstractMultiGridViewTest.java

```diff
@@ -199,4 +210,28 @@ public void testGlobalPreferences() {
         getView().loadListTable("key",
                                 consumer);
     }
+
+    @Test
+    public void testControlBulkOperationsEnabled() {
+
+        ListTable extendedPagedTable = mock(ListTable.class);
+        HasWidgets rightActionsToolbar = mock(HasWidgets.class);
+        boolean enabled = false;
+
+        final Button mockButton = GWT.create(Button.class);
+        ArrayList a = new ArrayList();
```

Comment:
@nmirasch you can use List<Button> buttonList = Collections.singletonList(mockButton); instead.